spec/forms/proceedings/substantive_defaults_form_spec.rb

```diff
@@ -28,9 +30,17 @@
           expect(proceeding.substantive_level_of_service).to eq 3
           expect(proceeding.substantive_level_of_service_name).to eq "Full Representation"
           expect(proceeding.substantive_level_of_service_stage).to eq 8
-          expect(proceeding.substantive_scope_limitation_code).to eq "FM062"
-          expect(proceeding.substantive_scope_limitation_meaning).to eq "Final hearing"
-          expect(proceeding.substantive_scope_limitation_description).to eq "Limited to all steps up to and including final hearing and any action necessary to implement (but not enforce) the order."
+        end
+
+        context "without calling the subject" do
+          let(:skip_subject) { true }
+
+          it "creates a scope_limitation object" do
+            expect { save_form }.to change(proceeding.scope_limitations, :count).by(1)
+            expect(proceeding.scope_limitations.where(scope_type: :substantive).first.code).to eq "AA019"
+            expect(proceeding.scope_limitations.where(scope_type: :substantive).first.meaning).to eq "Injunction FLA-to final hearing"
+            expect(proceeding.scope_limitations.where(scope_type: :substantive).first.description).to eq "As to proceedings under Part IV Family Law Act 1996 limited to all steps up to and including obtaining and serving a final order and in the event of breach leading to the exercise of a power of arrest to representation on the consideration of the breach by the court (but excluding applying for a warrant of arrest, if not attached, and representation in contempt proceedings)."
```

Comment:
[similarly](https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/4324/files#r977579513)